File: 06.py
#!/usr/bin/evn python3


def main():
    sum1 = 0
    for i in [ x*x for x in range(1, 101) ]:
        sum1 += i
#    print(sum1)

    sum2 = 0
    for i in [ x for x in range(1, 101) ]:
        sum2 += i

    sum2 = sum2 * sum2
    print(sum2 - sum1)


if __name__ == "__main__":
    main()

File: golang/06.go
package main

import (
	"fmt"
)

func main() {
	c := 100

	sum1 := 0
	sum2 := 0
	for i := 1; i <= c; i++ {
		sum1 += i * i
		sum2 += i
	}

	fmt.Println(sum2*sum2 - sum1)
}
